pod/playlist/views.py

```diff
@@ -154,7 +163,7 @@ def get_video_adv_note_list(request, video):
 
 
 def check_playlist_videos(playlist, data):
-    for slug in data:
+    for slug, position in data.items():
```

Comment:
ici position n'est pas utilisée, y a t-il une raison à cette modification ?
sinon, on peut faire ceci si flake8 n'aime pas `for slug in data:`:
```python
for slug in data.keys():
```

Reply:
Sinon on peut mettre un _ pour signifier que c’est une variable anonyme qui n’est pas utilisée ou en effet utiliser le keys